src/Jupyter/Extensions.cs

```diff
@@ -169,5 +169,9 @@ public static T WithStackTraceDisplay<T>(this T simulator, IChannel channel)
             };
             return simulator;
         }
+
+        internal static string TrimLeadingWhitespace(this string s) =>
+            new Regex(@"^\s+", RegexOptions.Multiline)
+                .Replace(s, string.Empty);
```

Comment:
can this just be string.TrimStart()?

Reply:
The main difference is that it trims whitespace from the start of each individual line, rather than from the string as a whole. Useful for formatting `@""` strings that are indented along with code, such as at https://github.com/microsoft/iqsharp/pull/114/files#diff-19b4af072f33fc9bfc7bda1e8a134d74R28. I'll add a comment to that effect, thanks!

Reply:
ah, got it! I see, that's what `RegexOptions.Multiline` means 👍